_targets.R

```diff
@@ -45,13 +48,50 @@ list(
       plot_site_data(out_file = state_plot_files,
                      site_data = nwis_data,
                      parameter = parameter),
-      format = "file")
+      format = "file",)
+)
+
+list(
+  # Identify oldest sites
+  tar_target(oldest_active_sites, find_oldest_sites(states, parameter)),
+
+  # per-state inventories, tallies, and plots
+  mapped_by_state_targets,
+
+  # combine the per-state tallies
+  tar_combine(obs_tallies,
+              mapped_by_state_targets[[3]],
```

Comment:
Just a note that you could also use `mapped_by_state_targets$site_tallies` here. It makes no difference functionally, but I prefer it because it allows me to see visually which of the mapped targets are being combined in this step, rather than having to remember which # is assigned to each of the mapped targets.

Reply:
Oh, great! That seems like it would be more resilient as well, since it won't break if someone changes the target numbering in my `tar_map()` by e.g. adding a target in the middle. I've switched to referring to the desired target by name, thanks!